src/backend/base/langflow/components/helpers/structured_output.py

```diff
@@ -17,24 +17,32 @@ class StructuredOutputComponent(Component):
         "or creating consistent outputs."
```

Comment:
this component is not working as expect when enable the tool_mode:

![image](https://github.com/user-attachments/assets/0a2bc52b-dffa-4ab8-b746-1b21201a17ff)

Reply:
This seems to be related to problems that are already fixed on main. I tried the exact same change and it works here. Also, make sure you're using a Language Model in the Structured Output.

![image](https://github.com/user-attachments/assets/48cd907b-47d1-4103-9457-547d1c43feb2)